python/tests/unit/arcticdb/version_store/test_empty_column_type.py

```diff
@@ -209,75 +209,63 @@ class TestCanAppendColumnWithNonesToColumn:
 
 
     def test_integer(self, lmdb_version_store_static_and_dynamic, int_dtype):
-        df_initial = pd.DataFrame({"col": np.array([1,2,3], dtype=int_dtype)})
+        df_initial = pd.DataFrame({"col": np.array([1,2,3], dtype=int_dtype), "other": [1,2,3]})
         lmdb_version_store_static_and_dynamic.write("sym", df_initial)
-        lmdb_version_store_static_and_dynamic.append("sym", pd.DataFrame({"col": [None, None]}))
-        assert_frame_equal(
-            lmdb_version_store_static_and_dynamic.read("sym").data,
-            pd.DataFrame({"col": np.array([1,2,3,0,0], dtype=int_dtype)})
-        )
-        assert_frame_equal(
-            lmdb_version_store_static_and_dynamic.read("sym", row_range=[0,3]).data,
-            df_initial
-        )
+        lmdb_version_store_static_and_dynamic.append("sym", pd.DataFrame({"col": [None, None], "other": [4,5]}))
+        expected_df = pd.DataFrame({"col": np.array([1,2,3,0,0], dtype=int_dtype), "other": [1,2,3,4,5]})
+        assert_frame_equal(lmdb_version_store_static_and_dynamic.read("sym").data, expected_df)
+        assert_frame_equal(lmdb_version_store_static_and_dynamic.read("sym", row_range=[0,3]).data, df_initial)
         assert_frame_equal(
             lmdb_version_store_static_and_dynamic.read("sym", row_range=[3,5]).data,
-            pd.DataFrame({"col": np.array([0,0], dtype=int_dtype)})
+            pd.DataFrame({"col": np.array([0,0], dtype=int_dtype), "other": [4, 5]})
         )
+        # Cannot compare with expected_df.tail(n=1) due to issue #1537: https://github.com/man-group/ArcticDB/issues/1537
+        expected_tail = pd.DataFrame({"col": np.array([0], dtype=int_dtype), "other": [5]})
+        assert_frame_equal(lmdb_version_store_static_and_dynamic.tail("sym", n=1).data, expected_tail)
```

Comment:
Tail and row range do the same thing under the hood here, so I don't think we need to test both (or if we do for some reason, should also test head)

Reply:
Nope, the row row range is performing a `direct_read` while tails is performing `read_and_process`. I was thinking that we should add separate test suite testing the process pipeline, so I did the minimal change here. Should I add head for completeness?

Reply:
You're right. Weird inconsistency, but yes, let's test `head` as well for completeness